modules/doobie-hikari/src/main/scala/pureconfig/module/doobie/hikari/package.scala

```diff
@@ -0,0 +1,17 @@
+package pureconfig.module.doobie
+
+import _root_.doobie.enumerated._
+import _root_.doobie.hikari._
+
+import pureconfig._
+import pureconfig.generic.semiauto._
```

Comment:
You cannot use semiauto here, as you'd force everyone using your module to use pureconfig-generic and shapeless. You'll need to use built-in `ConfigReader#forProduct` and `ConfigWrite#forProduct` calls.

Reply:
That's intentional, making these manually with the `forProduct` is just not feasible, the case class is huge.